bundles/com.zeligsoft.domain.omg.corba/models/CORBADomain.uml

```diff
@@ -537,7 +537,7 @@ self.oclAsType(uml::DataType).ownedAttribute->select(name = 'switchType')->first
           </ownedComment>
           <specification xmi:type="uml:OpaqueExpression" xmi:id="_hCQ1Ee1oEd2XOPM4ACfj4Q">
             <language>OCL</language>
-            <body>self.oclAsType(uml::NamedElement).name.matches('([a-zA-Z_])([a-zA-Z_0-9])*')</body>
+            <body>self.oclAsType(uml::NamedElement).name.matches('([a-zA-Z_])([a-zA-Z_0-9.])*')</body>
```

Comment:
This enables a dot ('.') to be part of a CORBA name other than in the first character. Is this intentional?